scripts/run_all_tests.sh

```diff
@@ -46,29 +46,42 @@ then
     echo "Successfully Downloaded all necessary vector test files"
 fi
 
+# Check if pgvector is available
+pgvector_installed=$($PSQL -U $DB_USER -c "SELECT 1 FROM pg_available_extensions WHERE name = 'vector'" -tA)
+
+# Generate schedule.txt
+rm -rf $TMP_OUTDIR/schedule.txt
+if [ -n "$FILTER" ]; then
+    if [[ "$pgvector_installed" == "1" ]]; then
+        TEST_FILES=$(cat schedule.txt | grep -E '^(test:|pgvector:)' | sed -e 's/^\(test:\|pgvector:\)//' | tr " " "\n" | sed -e '/^$/d')
```

Comment:
We could avoid all the shell wizardry by creating a separate schedule_pgvector.txt schedule with the single test and relying on pg_regress to do the parsing.
This works too though.